extensions/validation/src/test/java/org/mule/extension/validation/BasicValidationTestCase.java

```diff
@@ -149,22 +156,17 @@ public void isFalse() throws Exception
     public void notEmpty() throws Exception
     {
         final String flowName = "notEmpty";
-
```

Comment:
true that we should agree on something and always do it, although at least on the PR's I've review we always do it. If you want bring this up on post-scrum and we can hold the merge. 'Til then, I want my empty line :)